### PR TITLE
Use default OCP version on assisted-installer jobs

### PR DESCRIPTION
**Pull request by osherdp**
Change default release used on CI to the default version (currently 4.9).

### CODE REVIEW

File: ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh [hunk 1]
```diff
@@ -114,7 +114,7 @@ set -e
 
 if [ "${JOB_TYPE:-}" = "presubmit" ] && (( ! \${IS_REHEARSAL} )); then
   # We would like to keep running a stable version for PRs
-  echo "export OPENSHIFT_VERSION=4.8" >> /root/config
+  echo "export OPENSHIFT_VERSION=4.9" >> /root/config
```
Comment (eliorerz) [inline ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh:117]: Why do we need it also here? Assisted service define the default value.
I think we don't want to maintain the default values in multiple places (and repositories).

Comment (osherdp) [reply ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh:117]: I can try removing it
Probably a good idea

**Review by eliorerz — APPROVED** [review]
LGTM

**Comment by osherdp** [comment]
/unhold
seems okay I think

**Pull request merged**